app/controllers/albums_controller.rb

```diff
@@ -187,7 +187,7 @@ def update
   end  
 private
   def authenticate
```

Comment:
Cancan does this as part of load_and_authorize_resource. So it is already in ApplicationController. 

We do need to move the access rules to Ability though.